### PR TITLE
Reorganize docs

### PR DESCRIPTION
**Pull request by bohmt**
Description
Adds a two slab model that uses the UW2 workflow including geometry creation with Cubit/Trelis and material creation using PyNE.  This example is mostly script based.

Motivation and Context
This provides another example of DAGMC workflows

Changes
This adds another example plus updates some of the existing documentation.

Behavior
There was no existing tutorial on the UW2 workflow with Cubit/Trelis geometry creation and PyNE material creation.  Now there is.

Other Information
No other relevant information to this pull request.

Changelog file
There doesn't seem to be a changelog file.

### CODE REVIEW

**Review by NybergWISC — APPROVED** [review]
Looks good to me! I think it's a good idea to add more examples, especially including/referencing PyNE.

File: makeMatls.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+#
+# define iron for a shield
+ironvec={260000:1} # pure iron
+iron=Material()
+iron.density=7.86
+iron.from_atom_frac(ironvec)
+iron=iron.expand_elements()
+# to write out an mcnp material card
+iron.metadata['mat_number']=101
+iron.write_mcnp('twoslab_mcard.txt', 'atom')
```
Comment (gonuke) [inline makeMatls.py:15]: I think we can write this in one step now from the `MaterialLibrary`

File: twoslab.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  2. Create the two slab model and prepare it for DAGMC by running a Cubit/Trelis journal script
+  3. Run uwuw_preproc to inject materials into h5m faceted geometry file
+  4. Run DAG-MCNP62 using the supplied input file
+
+_Cubit is a meshing pre-processing software with CAD capabilities and Trelis is the commercially available version. It is used in this workflow to create the CAD model, assign materials to volumes, and finally export the model to the faceted DAGMC .h5m format._<br/></br>
+_Preparing and exporting the model to a faceted h5m format requires Cubit/Trelis and the DAGMC Cubit/Trelis plugin, which should be installed before starting the tutorial. Instructions are [here](https://svalinn.github.io/DAGMC/install/plugin.html)._
+
+## Procedure:
+
+### In the bash command shell type:
```
Comment (gonuke) [inline twoslab.md]: These headings should have titles that match the numbered list of steps above so learners can see where they are in the process

File: twoslab.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# Two Slab Model
+![viewshieldblock701](viewshieldblock701.png)<br/></br>
+The simple two slab model geometry consists of two slabs and a cube that holds an isotropic 14 MeV source.
+#### Goals of Example
```
Comment (gonuke) [inline twoslab.md]: This different heading levels have semantic meaning that we're kind of abusing...  should we just use bold here?

File: README.md [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
+**Purpose:**
 
-__These Tutorials use specific models to explain and demonstrate important steps of the DAGMC model workflow. By going through the tutorials one should understand all of the basic uses/functionality of SpaceClaim and Trelis/Cubit in the DAGMC workflow.__
+_These Tutorials use specific CAD models to explain and demonstrate important steps of the DAGMC workflow. By going through the tutorials one should understand the basic workflow of using DAGMC and the basic functionality of SpaceClaim and Trelis/Cubit in the DAGMC workflow._
 
-## Sections Sorted by Complexity of Model
-
-  * Overview of Tutorials:
+## Overview of Tutorials:
+  * Overview of tutorials that start with an existing CAD file:
```
Comment (gonuke) [inline README.md]:
```suggestion
  * Tutorials that start with an existing CAD file:
```

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks @bohmt - a few suggestions here

File: twoslab.md (hunk 3 above)
Comment (bohmt) [reply twoslab.md]: Yes, I got a bit sloppy with md syntax copying from a different document.

File: README.md (hunk 4 above)
Comment (bohmt) [reply README.md]: I will be implementing this and a few others that are similar.

File: twoslab.md (hunk 2 above)
Comment (bohmt) [reply twoslab.md]: Yes, that will be more consistent.

File: makeMatls.py (hunk 1 above)
Comment (bohmt) [reply makeMatls.py:15]: I know you can do this for a pyne Material but I didn't see this in the pyne documentation for a MaterialLibrary.   By the way, I assume you mean some function to write out the entire material library in MCNP mcard format to a file.

**Pull request merged**